src/code/audio/audio_init_params.c

```diff
@@ -1,12 +1,183 @@
 #include "global.h"
 
 const s16 gAudioTatumInit[] = {
-    0x1C00, // unused
-    0x30,   // gTatumsPerBeat
+    0x1C00,          // unused
+    TATUMS_PER_BEAT, // gTatumsPerBeat
 };
 
+// TODO: Extract from table?
+#define NUM_SOUNDFONTS 41
+#define SFX_SEQ_SIZE 0xC6A0
+#define AMBIENCE_SEQ_SIZE 0xFC0
+#define SOUNDFONT_0_SIZE 0x81C0
+#define SOUNDFONT_1_SIZE 0x36D0
+#define SOUNDFONT_2_SIZE 0xCE0
+
+// Sizes of everything on the init pool
+#define AI_BUFFERS_SIZE (AIBUF_SIZE * ARRAY_COUNT(gAudioContext.aiBuffers))
+#define SOUNDFONT_LIST_SIZE (NUM_SOUNDFONTS * sizeof(SoundFont))
+
+// 0x19BD0
+#define PERMANENT_POOL_SIZE \
+    (SFX_SEQ_SIZE + AMBIENCE_SEQ_SIZE + SOUNDFONT_0_SIZE + SOUNDFONT_1_SIZE + SOUNDFONT_2_SIZE + 0x430)
+
 const AudioHeapInitSizes gAudioHeapInitSizes = {
-    0x137F00, // heapSize
-    0x1C480,  // initPoolSize
-    0x1A000,  // permanentPoolSize
+    ALIGN16(sizeof(gAudioHeap) - 0x100),                                         // audio heap size
+    ALIGN16(PERMANENT_POOL_SIZE + AI_BUFFERS_SIZE + SOUNDFONT_LIST_SIZE + 0x40), // init pool size
+    ALIGN16(PERMANENT_POOL_SIZE),                                                // permanent pool size
+};
+
+#define REVERB_INDEX_0_SETTINGS \
+    { 1, 0x30, 0x3000, 0, 0, 0x7FFF, 0x0000, 0x0000, 0xFF, 0x3000, 0, 0 }
```

Comment:
Isn't the member for the 0xFF an s8, so this should be a -1? Actually further, it should probably be a macro since I'm guessing it's some kind of invalid case?